### PR TITLE
Add animation curve snapshot tests

### PR DESCRIPTION
**Pull request by lukebradford**
This adds snapshot tests for a handful of animation curves, with snapshots that show the curve drawn on a grid.

The code to draw the grid is broken out into a separate utility shared with the sample app.

### CODE REVIEW

File: Example/Unit Tests/AnimationCurveSnapshotTests.swift [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    func testCubicBezierEaseIn() {
+        verifyGraphView(for: CubicBezierAnimationCurve.easeIn)
+    }
+
+    func testCubicBezierEaseOut() {
+        verifyGraphView(for: CubicBezierAnimationCurve.easeIn)
+    }
+
+    func testCubicBezierEaseInEaseOut() {
+        verifyGraphView(for: CubicBezierAnimationCurve.easeIn)
```
Comment (NickEntin) [inline Example/Unit Tests/AnimationCurveSnapshotTests.swift]:
```suggestion
        verifyGraphView(for: CubicBezierAnimationCurve.easeInEaseOut)
```

File: Example/Unit Tests/AnimationCurveSnapshotTests.swift [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    func testSinusoidalEaseInEaseOut() {
+        verifyGraphView(for: SinusoidalEaseInEaseOutAnimationCurve())
+    }
+
+    func testCubicBezierEaseIn() {
+        verifyGraphView(for: CubicBezierAnimationCurve.easeIn)
+    }
+
+    func testCubicBezierEaseOut() {
+        verifyGraphView(for: CubicBezierAnimationCurve.easeIn)
```
Comment (NickEntin) [inline Example/Unit Tests/AnimationCurveSnapshotTests.swift]:
```suggestion
        verifyGraphView(for: CubicBezierAnimationCurve.easeOut)
```

File: Example/Unit Tests/AnimationCurveSnapshotTests.swift [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        containerView.backgroundColor = .white
+
+        let graphView: UIView = .init(frame: .init(x: margin, y: margin, width: graphSize, height: graphSize))
+        let gridLayer = makeGridLayer(frame: graphView.bounds)
+        graphView.layer.addSublayer(gridLayer)
+        let curveLayer = makeCurveLayer(frame: graphView.bounds, curve: curve)
+        graphView.layer.addSublayer(curveLayer)
+        containerView.addSubview(graphView)
+
+        FBSnapshotVerifyView(containerView)
```
Comment (NickEntin) [inline Example/Unit Tests/AnimationCurveSnapshotTests.swift]: Let's pass the file and line through here so any test failures show up on the right line.

**Review by NickEntin — COMMENTED** [review]
Looking good overall. Needs a couple updates to the tests, then rerecord snapshots and should be good to go. Thanks for doing this!

**Comment by NickEntin** [comment]
This will resolve #6

File: Example/Unit Tests/AnimationCurveSnapshotTests.swift (hunk 1 above)
Comment (lukebradford) [reply Example/Unit Tests/AnimationCurveSnapshotTests.swift]: Whoops! Thanks

**Pull request merged**